build.fsx

```diff
@@ -164,6 +164,24 @@ Target "AzureDocsDeploy" (fun _ ->
 Target "PublishDocs" DoNothing
 
 //--------------------------------------------------------------------------------
+// Build the SourceBrowser docs
+//--------------------------------------------------------------------------------
+Target "SourceBrowserDocs" <| (fun _ ->
+    let sourceBrowserDocsOutput = docDir @@ "SourceBrowser"
```

Comment:
Generating the sourcebrowser in the docs folder might interfere with the AzureDocsDeploy. It should be in its own folder "bin/sourcebrowser".